src/Common/src/System/Net/Logging/NetEventSource.Common.cs

```diff
@@ -13,6 +13,9 @@
 using System.Diagnostics.Tracing;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+#if NET46
```

Comment:
I thought it was useful to highlight where changes are NET46 specific. Otherwise, someone might try to remove that 'using' statement thinking it is not needed.